.vscode/settings.json

```diff
@@ -1,7 +1,6 @@
 {
-	"eslint.autoFixOnSave": true,
-	"eslint.validate": [
-		{ "language": "javascript", "autoFix": true },
-		{ "language": "html", "autoFix": true }
-	]
-}
+    "eslint.validate": [ "javascript" ],
+    "editor.codeActionsOnSave": {
+        "source.fixAll.eslint": true
+    }
+}
```

Comment:
Missing newline at the end of file

Reply:
Hmm I think vscode did this when it "helped" me to upgrade